docs/community/contributing.rst

```diff
@@ -58,6 +58,7 @@ install Batavia's JavaScript dependencies, and compile the Batavia library:
 .. code-block:: bash
 
     $ npm install
+    $ npm install -g npm
```

Comment:
Will do, I thought you had told me to do them in this order explicitly, but I may have mis-remembered.